### DISCUSSION TITLE
namespace 'wire' is not bound

### DISCUSSION THREAD

**Discussion by lperry65** [livewire/livewire#8004] [1. Help]
Just started using Livewire in PHPStorm and getting this warning, I tried install the Laravel plugin but that didn't help and I don't want to pay for a plugin as well as PHPStorm anyway.

Is there a way to fix this ?

Thanks,

Lee.

**Comment by joshhanley** [comment]
@lperry65 Can you screenshot the error?

**Reply by lperry65** [reply]
![image](https://github.com/livewire/livewire/assets/281669/a58ced92-9c82-4009-a611-494f4a3c9846)

**Reply by joshhanley** [reply]
@lperry65 ah right. Yeah I think it might need Laravel IDEA plugin to know how to handle Livewire stuff (not 100% though as I use VSCode).

**Reply by lperry65** [reply]
I installed the trial of the plugin with no change !

**Reply by joshhanley** [reply]
@lperry65 bugger 😕 I don't have PHPStorm anymore to test it out, so will have to see if someone who uses it can help.

**Reply by lperry65** [reply]
I just uninstalled the plugin restarted PHPStorm and reinstalled the plugin and it indexed and seems to have worked. But it's just the free trial so I'll have the same issue when it runs out. Thanks for your help though :)

**Reply by joshhanley** [reply]
@lperry65 ah nice (in that you got it working)! Yep, to me if you're going to pay for PHPStorm then it's worth paying for Laravel IDEA plugin as well as it really makes PHPStorm and Laravel work best together.
